src/en/authors-hook-debug.md

```diff
@@ -92,8 +92,10 @@ after the unit comes up, making it difficult to start a debug-hooks session in
 time to intercept them. If you're having difficulties, you can temporarily
 return an error code from your `install` hook (e.g. add an `exit 1` at the end
 of it), and start your session only when the unit reports an [error status
-](./authors-hook-errors.html). You should then run `juju resolved --retry` for
-the affected unit, and go back to the debug-hooks session to interact.
+](./authors-hook-errors.html). Juju version 2.0 and up will automatically retry
```

Comment:
thanks for that, I will add another issue to make sure this is captured in other relevant places